jax/experimental/jax2tf/jax_export.py

```diff
@@ -50,6 +51,40 @@
 
 DType = Any
 
+class DisabledSafetyCheck:
+  # Use a strings representation to aid human readability in serializations.
+  _impl: str
+
+  def __init__(self, _impl:str):
+    # Do not use directly, use builders `platform`, `custom_call`.
+    self._impl = _impl
+
+  def __str__(self):
+    return self._impl
+  __repr__ = __str__
+
+  def __eq__(self, other) -> bool:
+    return isinstance(other, DisabledSafetyCheck) and self._impl == other._impl
+
+  def __hash__(self) -> int:
+    return hash(self._impl)
+
+  @classmethod
+  def platform(cls) -> "DisabledSafetyCheck":
+    """Allows the execution platform to differ from the serialization platform."""
+    return DisabledSafetyCheck("platform")
+
+  @classmethod
+  def custom_call(cls, target_name: str) -> "DisabledSafetyCheck":
+    """Allows the serialization of a call target not known to be stable."""
+    return DisabledSafetyCheck(f"custom_call:{target_name}")
+
+  def is_custom_call(self) -> Optional[str]:
```

Comment:
How about `custom_call_target`? `is_custom_call` sounds like a method that will return a bool.